test/runtests.jl

```diff
@@ -95,6 +112,18 @@ end
         @test index_from_time(rate, t) == i
         @test time_from_index(rate, i) == t
     end
+
+    for rate in (101//2, 1001//10, 200, 256, 1, 10)
+        for sample_time in (Nanosecond(12345), Minute(5), Nanosecond(Minute(5)) + Nanosecond(1), Nanosecond(1), Nanosecond(10^6), Nanosecond(6970297031))
```

Comment:
Mighty long line ya got there, chief.
```suggestion
        for sample_time in (Nanosecond(12345), Minute(5), Nanosecond(Minute(5)) + Nanosecond(1),
                            Nanosecond(1), Nanosecond(10^6), Nanosecond(6970297031))
```